### DISCUSSION TITLE
Version mismatch?

### DISCUSSION THREAD

**Discussion by alex-w** [QtExcel/QXlsx#333] [Q&A]
The versions in cmake script and in release page are mismatch - is it correct?

**Comment by j2doll** [comment]
Dear @alex-w 
Sorry for late reply.
What part of the file are you talking about?

**Reply by alex-w** [reply]
Please check https://github.com/QtExcel/QXlsx/blob/0c8a17626f03b1863cdb6966b2108f4e0cf26a88/QXlsx/CMakeLists.txt#L6

**Reply by alex-w** [reply]
https://github.com/QtExcel/QXlsx/issues/321
